services/web/server/src/simcore_service_webserver/users/_handlers.py

```diff
@@ -202,3 +205,16 @@ async def mark_notification_as_read(request: web.Request) -> web.Response:
             raise web.HTTPNoContent(content_type=MIMETYPE_APPLICATION_JSON)
 
     raise web.HTTPNoContent(content_type=MIMETYPE_APPLICATION_JSON)
+
+
+@routes.get(f"/{API_VTAG}/me/permissions", name="list_user_permissions")
+@login_required
+@permission_required("user.permissions.read")
```

Comment:
Before you told me offline about the order of `@login_required` and `@permission_required` right?

Reply:
no. I spoke about the @routes
the other 2 are I think interchangeable, and anyway tested.